platforms/substrate/configuration/samples/network-sample-2.yaml

```diff
@@ -0,0 +1,173 @@
+##############################################################################################
```

Comment:
Can you update the network.yaml name to something which is more self-explanatory rather than just calling it `sample-2`?

Reply:
Updated to `substrate-network-config.yaml`. Thank you.